merlin-pipeline.py

```diff
@@ -29,10 +29,10 @@
 logger = logging.getLogger(__name__)
 
 args = None
-accelerator = 'nvidia-tesla-a100'
-node_pool = 'a100-pool'
-high_mem_node = 'high-mem-node'
-gcs_bucket_head = ''
+accelerator = 'nvidia-tesla-t4'
```

Comment:
Is it worth adding this as a command line argument (with this as a default) rather than hardcoded here?

Reply:
Done, thanks.